sentry/__manifest__.py

```diff
@@ -15,6 +15,7 @@
     'installable': True,
     'external_dependencies': {
         'python': [
+            'sentry_sdk',
```

Comment:
I think we could be added in the `author` section like "Vauxoo"

And you can add the key [`"maintainers": ["your-github-login"],`](https://github.com/OCA/maintainer-tools/blob/f1b559574648c01558b543e7550f5b9be7c92784/template/module/__manifest__.py#L13) to add original authors and we both

Reply:
Done

Reply:
Done